<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="entregada">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Potencia generada</string>
    </property>
   </widget>
   <widget class="QLabel" name="consumida">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Potencia consumida</string>
    </property>
   </widget>
   <widget class="QWidget" name="Color_blue" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="Color_red" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pmax1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Potencia máxima:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pmin1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Potencia minima:</string>
    </property>
   </widget>
   <widget class="QLabel" name="prom1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Promedio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pmax2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Potencia máxima:</string>
    </property>
   </widget>
   <widget class="QLabel" name="prom2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Promedio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pmin2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Potencia minima:</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>761</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
